src/Console/Ldap/SynchronizeUsersCommand.php

```diff
@@ -507,4 +566,69 @@ private function validateInput(InputInterface $input)
             }
         }
     }
+
+    /**
+     * Convert the old "single integer" format for the --deleted-user-strategy
+     * option into the new "3 comma-separated integers" format
+     *
+     * @deprecated
+     *
+     * @param int $deleted_user_strategy old format
+     *
+     * @return string new format
+     */
+    protected function convertOldDeletedUserStrategyToNew(int $deleted_user_strategy): string
+    {
+        Toolbox::deprecated("Usage of a deprecated format for the '--deleted-user-strategy' option.");
+        $this->output->writeln("<info>Warning: using deprecated --deleted-user-strategy format</info>");
+        $this->output->writeln('<info>Run "php bin/console ldap:synchronize --help for more details"</info>');
```

Comment:
Output should be translated. Also, command/options should be extracted from locale string to ensure translators will not alter them.
```suggestion
        $this->output->writeln('<info>' . sprintf(__('Warning: using deprecated %s format', '--deleted-user-strategy') . '</info>');
        $this->output->writeln('<info>' . sprintf(__('Run "%s" for more details, 'php bin/console ldap:synchronize --help') . '</info>');
```